rapthor/scripts/run_wsclean_mpi.sh

```diff
@@ -29,7 +30,11 @@ done
 
 # build the mpirun command
 infix=$(cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 6 | head -n 1)
-mpi_command="mpirun -np \$SLURM_JOB_NUM_NODES --pernode --prefix \$MPI_PREFIX -x LD_LIBRARY_PATH=\$LD_LIBRARY_PATH wsclean-mp -no-update-model-required -save-source-list -local-rms -join-channels -use-idg -mem ${mem} -j ${numthreads} -deconvolution-threads ${numdecthreads} -pol I -mgain 0.85 -log-time -fit-spectral-pol 3 -auto-threshold 1.0 -local-rms-window 50 -local-rms-method rms-with-min -aterm-kernel-size 32 -weight briggs ${robust} -name ${name} -fits-mask ${mask} -aterm-config ${config} -size ${imsize} -niter ${niter} -nmiter ${nmiter} -minuv-l ${min_uv_lambda} -maxuv-l ${max_uv_lambda} -scale ${cellsize_deg} -parallel-deconvolution 2048 -temp-dir ${dir_local} -channels-out ${channels_out} -deconvolution-channels ${deconvolution_channels} -taper-gaussian ${taper_arcsec} -auto-mask ${auto_mask} -idg-mode ${idg_mode} ${msin}"
+if [ -n "${multiscale_scales_pixel}" ]; then
+    mpi_command="mpirun -np \$SLURM_JOB_NUM_NODES --pernode --prefix \$MPI_PREFIX -x LD_LIBRARY_PATH=\$LD_LIBRARY_PATH wsclean-mp -no-update-model-required -multiscale -save-source-list -local-rms -join-channels -use-idg -mem ${mem} -j ${numthreads} -deconvolution-threads ${numdecthreads} -pol I -mgain 0.85 -log-time -fit-spectral-pol 3 -multiscale-shape gaussian -auto-threshold 1.0 -local-rms-window 50 -local-rms-method rms-with-min -aterm-kernel-size 32 -weight briggs ${robust} -name ${name} -fits-mask ${mask} -aterm-config ${config} -size ${imsize} -niter ${niter} -nmiter ${nmiter} -minuv-l ${min_uv_lambda} -maxuv-l ${max_uv_lambda} -scale ${cellsize_deg} -multiscale-scales ${multiscale_scales_pixel} -parallel-deconvolution 2048 -temp-dir ${dir_local} -channels-out ${channels_out} -deconvolution-channels ${deconvolution_channels} -taper-gaussian ${taper_arcsec} -auto-mask ${auto_mask} -idg-mode ${idg_mode} ${msin}"
+else
+    mpi_command="mpirun -np \$SLURM_JOB_NUM_NODES --pernode --prefix \$MPI_PREFIX -x LD_LIBRARY_PATH=\$LD_LIBRARY_PATH wsclean-mp -no-update-model-required -save-source-list -local-rms -join-channels -use-idg -mem ${mem} -j ${numthreads} -deconvolution-threads ${numdecthreads} -pol I -mgain 0.85 -log-time -fit-spectral-pol 3 -auto-threshold 1.0 -local-rms-window 50 -local-rms-method rms-with-min -aterm-kernel-size 32 -weight briggs ${robust} -name ${name} -fits-mask ${mask} -aterm-config ${config} -size ${imsize} -niter ${niter} -nmiter ${nmiter} -minuv-l ${min_uv_lambda} -maxuv-l ${max_uv_lambda} -scale ${cellsize_deg} -parallel-deconvolution 2048 -temp-dir ${dir_local} -channels-out ${channels_out} -deconvolution-channels ${deconvolution_channels} -taper-gaussian ${taper_arcsec} -auto-mask ${auto_mask} -idg-mode ${idg_mode} ${msin}"
```

Comment:
OK, in principle. Maybe it would be nicer, for readability, to have a base non-multiscale command, which is then extended with the extra multiscale options.

Reply:
Good point -- I didn't think of that (I just copied and pasted the command from the multiscale script). Anyway, hopefully the scripts will get replaced sometime soon with proper Toil integration